Jenkinsfile

```diff
@@ -1,17 +1,22 @@
 pipeline {
-  agent any
+  agent none
   stages {
-    stage('') {
-      agent {
-        node {
-          label 'linux'
+    stage('test') {
+      parallel {
+        stage('python3') {
+          agent {
+            dockerfile {
+              // Large image with full OpenModelica build dependencies; lacks omc and OMPython
+              label 'linux'
+              dir '.jenkins/python3'
+              additionalBuildArgs '--pull'
+            }
+          }
+          steps {
+            sh 'pytest testing'
+          }
         }
-
-      }
-      steps {
-        sh 'hostname'
```

Comment:
The hostname command is pretty useful when you need to figure out which damn machine ran the command :D